tests/src/system/basic/WskBasicTests.scala

```diff
@@ -338,6 +338,20 @@ class WskBasicTests
             }
     }
 
+    /**
+      * Tests creating an nodejs action that throws a whisk.error() response. The error message thrown by the
+      * whisk.error() should be returned.
+      */
+    it should "create and invoke a blocking action resulting in a whisk.error response" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "whiskError"
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(TestUtils.getTestActionFilename("applicationError1.js")))
+            }
+            wsk.action.invoke(name, blocking = true, expectedExitCode = 246)
+              .stdout should include regex (""""error": "This error thrown on purpose by the action."""")
```

Comment:
I'd drop off the "error:" part of the regex pattern.  This is CLI error formatting stuff and could vary.  The key test is whether or not the argument to whisk.error() was displayed.